src/components/bc-start.ts

```diff
@@ -60,7 +76,7 @@ export class Start extends withTwind()(BitcoinConnectElement) {
               How would you like to connect?
             </h1>
 
-            <bc-connector-list />
+            <bc-connector-list></bc-connector-list>
```

Comment:
thanks for fixing this too :pray:

Reply:
I saw this when I ran `npm run lint`. The other error was in the test-file for `my-component`. I was not sure what do there hence did not fixed that.